gateway/Dockerfile

```
@@ -1,28 +1,31 @@
 
 COPY gradlew .
 COPY gradle gradle
-COPY settings.gradle .
 COPY auth-shared auth-shared
 COPY gateway gateway
 
+RUN echo "include 'gateway'" > settings.gradle && \
+    echo "include 'auth-shared'" >> settings.gradle
+
 RUN chmod +x gradlew
 RUN ./gradlew :gateway:clean :gateway:build -x test --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)